### ISSUE TITLE
cpplint not work in windows 10

### ISSUE THREAD

**Issue by mumin16** [QiuminGe/cpp-check-lint#16]
cpplint not work in windows 10. cppcheck is ok. both are ok in ubuntu

**Comment by mumin16** [comment]
ok ok ok. we give full path

**Comment by QiuminGe** [comment]
Later, I'll test how the built-in binaries work in win10. I use win7 and Linux

**Comment by QiuminGe** [comment]
* Executable file selection logic    
 ``` 
    if (cppcheck configure is null) {
        use builtin binaries
    } else {
        if( ("path to executable" --version).trim().toLowerCase().startsWith("cppcheck") ){
            use "path to executable"
        } else {
           use builtin binaries 
        }
    }

    if (cpplint configure is null) {
        use builtin binaries
    } else {
        if("path to executable"){
            use "path to executable"
        } else {
           use builtin binaries 
        }
    } 
```

**Comment by QiuminGe** [comment]
please try v1.3.6

**Comment by QiuminGe** [comment]
please try v1.3.6

**Comment by QiuminGe** [comment]
windows builtin binaries fixed at v1.4.6
